lib/sqlbuilder/builder.go

```diff
@@ -268,10 +272,19 @@ func Map(item interface{}, options *MapOptions) ([]string, []interface{}, error)
 
 		for _, fi := range fieldMap {
 
+			// Check for deprecated tags and give suggestions on how to fix them.
+			deprecatedTags := map[string]string{
+				"stringarray": "postgresql.StringArray",
+				"int64array":  "postgresql.Int64Array",
+			}
+			for k, v := range deprecatedTags {
```

Comment:
`lib/pq` handles this case just fine for binary and text node, if we use their types. It doesn't work if we use our own encoding, so we'll have to deprecate this and fail at this point.